src/bun.js/RuntimeTranspilerCache.zig

```diff
@@ -382,10 +382,10 @@ pub const RuntimeTranspilerCache = struct {
 
     fn reallyGetCacheDir(buf: *bun.PathBuffer) [:0]const u8 {
         if (comptime bun.Environment.isDebug) {
-            bun_debug_restore_from_cache = bun.getenvZ("BUN_DEBUG_ENABLE_RESTORE_FROM_TRANSPILER_CACHE") != null;
+            bun_debug_restore_from_cache = bun.EnvVar.bun_debug_enable_restore_from_transpiler_cache.get();
```

Comment:
This went from a string to a bool. Seems like the original intent is for it to be a bool